### DISCUSSION TITLE
I2C sensor detected but no values HTU21D on ESP32

### DISCUSSION THREAD

**Discussion by horsey2022** [arendst/Tasmota#19498] [Q&A]
Manual scan of The temperature sensor HTU21D is detected but no values are measured. on a ESP32 running the standard pre-compiled sensor version of 13.1.0.
Tasmota does not display the sensor values, any ideas of what could be wrong?

> 13:43:57.901 CMD: I2CScan
13:43:57.904 SRC: WebConsole from 192.168.0.3
13:43:57.905 CMD: Grp 0, Cmd 'I2CSCAN', Idx 1, Len 0, Pld -99, Data ''
13:43:57.931 MQT: stat/tasmota_EEE4C4/RESULT = {"I2CScan":"Device(s) found on bus1 at 0x40"}

**Comment by sfromis** [comment]
There exists no "standard pre-compiled sensor version" for ESP32. Equivalent sensors should be included in `tasmota32.bin`

If the sensor driver is included in your binary, number 9 should appear in the output of the command `I2CDriver`

**Reply by horsey2022** [reply]
I used the web install and there is an option for sensors. see the screenshot
![image](https://github.com/arendst/Tasmota/assets/99337610/1d6db4a9-497d-48ca-8cf8-6e95576d11f4)

Here is the info from the  drivers
00:48:49.797 CMD: I2cDriver
00:48:49.803 RSL: RESULT = {"I2CDriver":"7,8,9,10,11,12,13,14,15,17,18,20,24,29,31,36,41,42,44,46,48,58,62,65,69,76,77,82"}

**Reply by horsey2022** [reply]
Solved. It was a Hardware problem.
The SCL and SDA were connected by a ribbon cable with the two next to each other. When I separated the wires connecting the SCL and SDA things just started working.
